firmware/brake/kia_soul_ps/brake_control_module.ino

```diff
@@ -424,23 +427,25 @@ void close_master_cylinder_solenoid( )
 // *****************************************************
 void check_driver_brake_override( )
 {
-	master_cylinder_solenoid._pressure1 = convert_to_voltage( analogRead( master_cylinder_solenoid._sensor1_pin ) );
-	master_cylinder_solenoid._pressure2 = convert_to_voltage( analogRead( master_cylinder_solenoid._sensor2_pin ) );
-
-	//if current pedal pressure is greater than the limit
-	//(because of driver override by pressing the brake pedal), disable.
-	if( ( master_cylinder_solenoid._pressure1 > PEDAL_THRESHOLD ) ||
-		( master_cylinder_solenoid._pressure2 > PEDAL_THRESHOLD ) )
-	{
-		DEBUG_PRINT( "Brake Pedal Detected" );
-		pressure_req = ZERO_PRESSURE;
-		local_override = 1;
-		brake_state_machine.transitionTo( Wait );
-	}
-	else
-	{
-		local_override = 0;
-	}
+    int16_t read1 = analogRead( master_cylinder_solenoid.sensor1_pin );
+    master_cylinder_solenoid.pressure1 = convert_ADC_reading_to_voltage( read1 );
```

Comment:
change name to convert_(ADC?)_reading_to_pressure, now that I look at it, I like the ADC in there.